src/main/java/org/truffleruby/language/methods/ModuleBodyDefinitionNode.java

```diff
@@ -74,7 +74,13 @@ private LexicalScope prepareLexicalScope(LexicalScope staticLexicalScope, Lexica
         } else {
             // Cache the scope per module in case the module body is run multiple times.
             // This allows dynamic constant lookup to cache better.
-            return lexicalScopes.computeIfAbsent(module, m -> new LexicalScope(parentLexicalScope, module));
+            LexicalScope scope = lexicalScopes.get(module);
```

Comment:
No, I was just trying out different variants of the computeIfAbsent replacement, one of which reassigned the local.
I can make it final.